src/integrations/contact-input-custom-chip-component.test.tsx

```diff
@@ -191,14 +246,14 @@ describe('Contact input custom chip component', () => {
 		const user5 = screen.getByText(user5Mail);
 		const user6 = screen.getByText(user6Mail);
 
-		expect(user4).toBeInTheDocument();
-		expect(user5).toBeInTheDocument();
-		expect(user6).toBeInTheDocument();
+		expect(user4).toBeVisible();
+		expect(user5).toBeVisible();
+		expect(user6).toBeVisible();
 
-		expect(showMore).not.toBeInTheDocument();
+		expect(showMore).not.toBeVisible();
```

Comment:
not.toBeInTheDocument was right here, it is a full check that the element is not even in the dom

Reply:
You are right, I made a find and replace and didn t notice this